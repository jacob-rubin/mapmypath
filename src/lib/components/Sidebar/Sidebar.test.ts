// It starts out collapsed

// When button clicked, it opens

// When button clicked again, it closes

// The text should slide out from the right

// It should span the whole height of the screen
