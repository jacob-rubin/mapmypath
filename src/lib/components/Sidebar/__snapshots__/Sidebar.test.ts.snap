// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Sidebar > matches the snapshot 1`] = `"<div class="flex h-screen w-72 flex-grow flex-row items-center py-2"><div class="card card-normal h-full w-full overflow-auto bg-neutral-content p-2" data-testid="sidebar"></div><!----> <div class="tooltip tooltip-right flex" data-testid="tooltip" data-tip="Collapse"><button class="h-10 rounded-l-none rounded-r-lg border-none bg-neutral-content"><svg viewBox="0 0 24 24" width="1.2em" height="1.2em" data-testid="collapse" class="h-6 w-6"><path fill="currentColor" d="M15.41 16.58L10.83 12l4.58-4.59L14 6l-6 6l6 6z"></path></svg><!----></button></div></div>"`;
